docs/content/en/docs-dev/operator-manual/piped/adding-helm-chart-registry.md

```diff
@@ -0,0 +1,24 @@
+---
+title: "Adding a helm chart registry"
+linkTitle: "Adding helm chart registry"
+weight: 6
+description: >
+  This page describes how to add a new Helm chart registry.
+---
```

Comment:
I feel that this should be merged into the current "Adding Helm chart repository".
We will have a new page "Adding helm repository or registry" to contain both of the contents.
WDYT?

Reply:
Since the chart registry chapter is shorter than others, I think no problem integrating it into the chart repository page.
cc: @khanhtc1202 @knanao

Reply:
@nghialv merged these docs with this commit: [4485be1](https://github.com/pipe-cd/pipecd/pull/3643/commits/4485be1321be9d7256767c4174c3743e03345e82)